### PR TITLE
feat(rest-client): add client-endpoint, and bump-version generators

### PR DESCRIPTION
**Pull request by nandorszentpeteri**

### CODE REVIEW

File: e2e/rest-client-e2e/tests/bump-version.spec.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+describe('http-client e2e', () => {
+    beforeAll(async () => {
+        await newProject(['@ensono-stacks/rest-client'], ['@nrwl/js']);
+    });
+
+    afterAll(() => {
+        cleanup();
+    });
+
+    describe('client-endpoint', () => {
```
Comment (cloudratha) [inline e2e/rest-client-e2e/tests/bump-version.spec.ts:18]: Creating a new Project is expensive. could we just have these as separate tests in one spec file?

before: create new Project
test1: create client
test2: create endpoint
test3: bump version endpoint

What do you think?

Comment (nandorszentpeteri) [reply e2e/rest-client-e2e/tests/bump-version.spec.ts:18]: I agree actually, it ruins a separation of concern which I like to introduce in my files but it makes sense.

File: packages/rest-client/src/generators/client-endpoint/files/index.ts__template__ [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+<% if (methods.includes('put')) { %>
+export const put<%= className %> = async (data, config): Response<<%= className %>> => 
+    await httpClient.put(API_ENDPOINT, data, config)
+<% } %>
+
+<% if (methods.includes('patch')) { %>
+export const patch<%= className %> = async (data, params): Response<<%= className %>> => 
+    await httpClient.patch(API_ENDPOINT, data, config)
+<% } %>
```
Comment (andreyroth) [inline packages/rest-client/src/generators/client-endpoint/files/index.ts__template__:41]: is the lack of newline here on purpose?

File: packages/rest-client/src/generators/client-endpoint/files/index.types.ts__template__ [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export interface <%= className %> {
+
+}
+
+export interface <%= className %>Data {
+
+}
```
Comment (andreyroth) [inline packages/rest-client/src/generators/client-endpoint/files/index.types.ts__template__:7]: is the lack of newline here on purpose?

**Review by andreyroth — APPROVED** [review]

**Review by cloudratha — APPROVED** [review]

**Pull request merged**